### PR TITLE
Register Subdomain: jorge.cribb

### PR DESCRIPTION
**Pull request by jrcribb**
Added the domain: jorge.cribb

### CODE REVIEW

File: domains/jrcribb.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "description": "JRCribb's Work",
+  "owner": {
+    "username": "jrcribb",
+    "email": "jorge.cribb@gmail.com"
+  },
+  "record": {
+    "CNAME": "jrcribb.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/jrcribb.json:8]: 404?

File: domains/jorge.cribb.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (WilliamDavidHarrison) [inline domains/jorge.cribb.json:1]: Please register `cribb.is-a.dev` before you register `jorge.cribb.is-a.dev`

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]

**Review by wdhdev — CHANGES_REQUESTED** [review]
You need to register `cribb.is-a.dev` before you register `jorge.cribb.is-a.dev`

File: domains/jorge.cribb.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    {
+      "owner": {
+        "username": "jrcribb",
+        "email": "jorge.cribb@gmail.com"
+      },
+
+      "record": {
+        "A": ["dev.coraus.com.ar:2080"]
```
Comment (WilliamDavidHarrison) [inline domains/jorge.cribb.json:9]: Invalid A record

File: domains/jrcribb.json (hunk 1 above)
Comment (WilliamDavidHarrison) [inline domains/jrcribb.json:8]: 404

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**